scripts/readme.md

```diff
@@ -0,0 +1,37 @@
+```shell
+# To login
```

Comment:
use multiple shell code blocks with text in between so it can be read more easily
ie 

Once a job is submitted, it can be viewed and modified using squeue and scancel
```shell
# To see all queue
$ squeue
# To see my queue
$ squeue -u ll720
# To remove a job
$ scancel job_id
```